lib/iso_builder.py

```diff
@@ -42,6 +42,8 @@ def __init__(self, config):
         (_, _, self.arch) = distro_utils.detect_distribution()
         self.mock_binary = self.common_config.get('mock_binary')
         self.mock_args = self.config.get('mock_args') or ""
+        self.pungi_binary = self.common_config.get('pungi_binary') or "pungi"
```

Comment:
I don't think this "default" should be here. I think we should fail if the value is not present in the config file or command line. You're just mimicking the behavior above though, so let's leave this discussion for another time. Alan's PR might fix this.